modules/votes/ui/VoteActionsModals/SuccessModal/VoteSuccessModalStyle.tsx

```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components'
+
+export const VoteAsDelegateWrap = styled.div``
```

Comment:
Do not forget to fill up or remove this boilerplate here

Reply:
fixeed in 4604a017354cc8efeeff5ce738df3a4613a574df